util/http_utils.go

```diff
@@ -0,0 +1,83 @@
+package util
+
+import (
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+)
+
+func WriteError(w http.ResponseWriter, context, err string) {
+	_, writeErr := w.Write([]byte(err + "\n"))
+	if writeErr != nil {
+		logrus.Errorf(WritingResponseErr, writeErr)
+	}
+	logrus.Errorf(context, err)
+}
+
+func getClaimFile(w http.ResponseWriter, r *http.Request) multipart.File {
+	err := r.ParseMultipartForm(ParseLowerBound << ParseUpperBound)
+	if err != nil {
+		WriteError(w, RequestContentTypeErr, err.Error())
+		return nil
+	}
+
+	claimFile, _, err := r.FormFile(ClaimFileInputName)
+	if err != nil {
+		WriteError(w, FormFileErr, err.Error())
+		return nil
+	}
+	return claimFile
+}
+
+func ReadClaimFile(w http.ResponseWriter, r *http.Request) []byte {
+	claimFile := getClaimFile(w, r)
+	defer claimFile.Close()
+
+	claimFileBytes, err := io.ReadAll(claimFile)
+	if err != nil {
+		WriteError(w, ReadingFileErr, err.Error())
+		return nil
+	}
+
+	return claimFileBytes
+}
+
+func getEnv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return fallback
+}
+
+func GetDatabaseEnvVars() (user, password, dbUrl, port string) {
+	user = getEnv("DB_USER", "admin")
+	password = getEnv("DB_PASSWORD", "collector1")
```

Comment:
Do we want to store here our password as is? this way any one can get the entire database..

Reply:
No, these are for local testing purpose. We store password with base64 encoded in the database.

Reply:
This is the password of the mysql instance it self. so using it and the given host one can connect to the db it self using mysql server and that way access the tables.